### ISSUE TITLE
1049 / 5000 Çeviri sonuçları When I enter the code on the certificate verification page, only the name and course name are written, no other fields

### ISSUE THREAD

**Issue by alihamas575** [mdjnelson/moodle-mod_customcert#424]
![image](https://user-images.githubusercontent.com/54097193/120085900-ca0f2a00-c0e4-11eb-8c80-c80e8b503e05.png)
All information added to the certificate should be included in the certificate validation field, user-defined fields, date, training period, completed activities, etc.

In the certificate verification field, there is only the phrase "verified". Instead, it is better to see either the whole certificate or the fields in the certificate.

This can be bypassed with a scam.
We have a training with a time limit, the student takes the certificate, takes a sample of the certificate with the help of a browser, and changes the training deadline or other information, so that he changes the document in his hand, but when the certificate looks at the verification, the verification code is successful, so the certificate appears to be correct.
Because there is no date in the certificate verification field, but we added this date in the certificate, only the course name and student name appear on the verification screen, no other information.

This will not be correct information. Instead, you can view the full image of the certificate or all fields in the certificate.

**Comment by alihamas575** [comment]
Is the following information sufficient to explain the subject more clearly?
Hello,
When we enter the certificate verification code on the certificate verification page, it shows a warning that only the certificate is valid. This may lead to a misunderstanding.

For example, Andrew user completed the course with the certificate number XXXXXXX01 with C1 Level in English, before giving this certificate to the institution x, he opened the pdf file with a free program such as pdf exchange editor + and changed the place from C1 to A1.

The human resources manager of the workplace where he issued the certificate, Michael, wrote the certificate number XXXXXXX01 on the certificate verification screen on my website, which is under my management, and the room says "successful". But my student named Andrew changed the success of the C1 course written in the Certificate and made A1. As the site administrator, I am not aware of this and unfortunately I gave Michael wrong information.

More fields are required in the certificate validation field than just verified,

How many hours of the course, what certificate did he get, the title of the certificate etc. In fact, if possible, it would be a better approach to show the same on the site as what is written in the certificate pdf document.

Otherwise, it becomes a document that can be manipulated and misled people.

**Comment by mdjnelson** [comment]
Yep, I understand the issue. I just need to find the time to work on this. Thanks for the report.

**Comment by alihamas575** [comment]
thank you very much, see you

**Comment by mdjnelson** [comment]
Closing due to inactivity as part of issue housekeeping. I’m a solo maintainer with limited time right now, so I’m focusing on issues with recent activity and clear next steps.

If this is still relevant, please comment with an update (bug: current repro steps + environment; request: use case + desired outcome). With updated info, I can re-triage and reopen or track it in a new issue if needed.
